setup.py

```diff
@@ -40,6 +41,7 @@
                       'pyelasticsearch>=1.1,<2.0',
                       'Pygments>=2.0.1,<3.0',
                       'python-hglib>=1.6,<2.0',
+                      'xpidl>=1.0',
```

Comment:
1. This isn't in PyPI yet, so listing it here will just cause 404s. Let's upload it to PyPI.
2. We'll also need to add this to requirements.txt so it will be pinned to a hash.